firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/SessionReportingCoordinator.java

```diff
@@ -289,6 +300,19 @@ private CrashlyticsReport.Session.Event addLogsAndCustomKeysToEvent(
     return eventBuilder.build();
   }
 
+  private CrashlyticsReport.Session.Event addRolloutsStateToEvent(
+      CrashlyticsReport.Session.Event capturedEvent, UserMetadata reportMetadata) {
+    final CrashlyticsReport.Session.Event.Builder eventBuilder = capturedEvent.toBuilder();
+    final List<CrashlyticsReport.Session.Event.RolloutAssignment> reportRolloutsState =
+        reportMetadata.getRolloutsState();
+    if (reportRolloutsState.size() > 0) {
```

Comment:
Small style suggestion: I like to have my functions return early in the unhappy path (or less-expected path), and return at the end in the happy path. What do you think about reversing this and checking `(....size() == 0) { ... return capturedEvent }`?

Reply:
done!